unit-tests/test-fw-update.py

```diff
@@ -183,12 +183,9 @@ def find_image_or_exit( product_name, fw_version_regex = r'(\d+\.){3}(\d+)' ):
 if device.is_update_device():
     log.d( "recovering device ..." )
     try:
-        image_file = find_image_or_exit(product_name) if not custom_fw_d400_path else custom_fw_d400_path
+        # image_file = find_image_or_exit(product_name) if not custom_fw_d400_path else custom_fw_d400_path
+        image_file = find_image_or_exit(product_name) 
         cmd = [fw_updater_exe, '-r', '-f', image_file]
-        if custom_fw_d400_path:
-            # Add '-u' only if the path doesn't include 'signed'
-            if 'signed' not in custom_fw_d400_path.lower():
-                cmd.insert(1, '-u')
```

Comment:
It seems this will work, but the if the user assigns some unsigned fw to param custom_fw_d400, he will assume then that the tests are done with this fw.
Maybe better to first burn the signed fw, to get rid of the recovery issue, and then to burn the unsigned fw?